### ISSUE TITLE
Inject Inspector shortcut is not friendly to international keyboard configurations

### ISSUE THREAD

**Issue by tiagosomda** [aframevr/aframe-inspector#583]
> **Note:** when I say "international keyboard layouts" I mean keyboards that are firing an [Input Method Editor ](https://www.w3.org/TR/ime-api/) event (composing a key) when pressing the key combination `ctrl + alt + i`

Some International keyboard layouts when given the key combination `ctrl + alt + i` will fire an event that is not recognized by the current code in aframe inspector.

code reference: [src/index.js line 180](https://github.com/aframevr/aframe-inspector/search?q=var+shortcutPressed+%3D+evt.keyCode+%3D%3D%3D+73+%26%26+evt.ctrlKey+%26%26+evt.altKey%3B&unscoped_q=var+shortcutPressed+%3D+evt.keyCode+%3D%3D%3D+73+%26%26+evt.ctrlKey+%26%26+evt.altKey%3B)

In an international keyboard,  
when the 'i' key is pressed, the event has the values:
- evt.ctrlKey : false
- and evt.altKey : false
  
This makes the current check fail :  
`var shortcutPressed = evt.keyCode === 73 && evt.ctrlKey && evt.altKey;`  
  
Thus, to be more friendly to those types of keyboard layouts,  
I believe that we should change the code to handle this edge case.

here is a proposed solution:

```
...
  _this3.toggleShortcutStarted = false
  window.addEventListener('keydown', function (evt) {
    // Alt + Ctrl + i: Shorcut to toggle the inspector
    if (_this3.toggleShortcutStarted === false && evt.ctrlKey && evt.altKey) {
      _this3.toggleShortcutStarted = true;
    } else if (_this3.toggleShortcutStarted === true && evt.keyCode === 73) {
      _this3.toggleShortcutStarted = false;
      _this3.toggle();
    }
  });
...
``` 

There is probably a more suitable place for the  `toggleShortcutStarted` value.  
I just used `_this3` as an example.

Adding @dmarcos who I saw that was answering people about this issue on slack.

**Comment by vincentfretin** [comment]
I think this is fixed by #658 you can probably close @dmarcos

**Comment by dmarcos** [comment]
Fixed by https://github.com/aframevr/aframe-inspector/pull/658
